futures-util/src/compat/executor.rs

```diff
@@ -0,0 +1,47 @@
+
+pub trait ExecCompat: Executor01<
+        Compat<FutureObj<'static, ()>, BoxedExecutor>
+    > + Clone + Send + 'static
+{
+    fn compat(self) -> ExecutorCompat<Self> 
```

Comment:
To be consistent that should be in `ExecutorExt` in `futures_util/task/executor.rs` (doesn't exist yet, but `ContextExt` does, so `ExecutorExt` should as well)